libs/dal/src/repositories/localization/localization.schema.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose, { Schema } from 'mongoose';
+import { schemaOptions } from '../schema-default.options';
+import { LocalizationDBModel } from './localization.entity';
+
+const localizationSchema = new Schema<LocalizationDBModel>(
+  {
+    locale: {
+      type: Schema.Types.String,
+      required: true,
+    },
+    content: {
+      type: Schema.Types.String,
+      required: true,
+    },
+    _localizationGroupId: {
+      type: Schema.Types.ObjectId,
+      ref: 'LocalizationGroup',
+      required: true,
+    },
+    _environmentId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Environment',
+      required: true,
+    },
+    _organizationId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Organization',
+      required: true,
+    },
+  },
+  schemaOptions
+);
```

Comment:
Would need to ensure we add indexes for acess patterns on both new colletions in prod and stagng